### DISCUSSION TITLE
Is there a way to export raw data of cell population frenquencies?

### DISCUSSION THREAD

**Discussion by JingAnyaSun** [github/catalyst#277] [General]
Hi, 

There is a function in CATALYST called plotAbundances to plot the population abundances after clustering. I wonder if there is a way to export the raw frequency data of each cell population in each sample? Thanks!

Best,
Jing

**Comment by keithamus** [comment]
Hey @JingAnyaSun! I think the repository you're after is https://github.com/HelenaLC/CATALYST. This is the discussions board for a front-end UI framework that has a similar name.
